lib/_sendEvent.ts

```diff
@@ -1,4 +1,5 @@
 import { isNumber, isUndefined, isString, isFunction } from "./utils/index";
+import { version } from "../package.json";
```

Comment:
Why this import here? (`version` does not seems to be used in this file)

Reply:
it will be, but it makes no sense at this stage you're right.
Removing